src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -359,6 +378,13 @@ export const ArtworkAppFragmentContainer = createFragmentContainer(
           internalID
           slug
           extendedBiddingIntervalMinutes
+          isClosed
```

Comment:
I think we can safely remove the `isClosed` from here don't think we use it anywhere
```suggestion
```

Reply:
This actually is needed by the `lotIsClosed` helper - I'll push an update to this PR that updates the typing on that helper to make it clear (and will cause a TS error without `isClosed`).